sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt

```diff
@@ -231,7 +231,8 @@ internal class RadarLocationManager(
         }
     }
 
-    internal fun updateTrackingFromMeta(meta: RadarMeta?) {
+    internal fun updateTrackingFromMeta(status: RadarStatus, meta: RadarMeta?) {
```

Comment:
hmm why does this fn need to know about `status`? doesn't feel right to pass this through just for a log line